CHANGES.md

```diff
@@ -5,6 +5,8 @@ Change Log
 
 * Pin `html-to-react` to `1.4.5` due to ESM module in dependency (`parse5`) breaking webpack
 * Add step to `"Deploy TerriaMap"` action to save `yarn.lock` after `sync-dependencies` (for debug purposes)
+* Upgrade i18next to `v21`
+* Update `translation` to use new pluralization format (i18next JSON v4)
```

Comment:
Doesnt the i18next.init() in this PR specify to still use v3? https://github.com/zoran995/terriajs-i18n/blob/cf5d7a1ff1043e1fbbd1e26dcf4392c3cdb2cd96/lib/Models/Internationalization.ts#L86

Reply:
Forgot to revert this in changes.md, we have to wait with upgrade of files to v4 until weblate supports them